conda/recipes/rapids-xgboost/meta.yaml

```diff
@@ -28,13 +28,14 @@ requirements:
     - python
     - cuda-version ={{ cuda_version }}
   run:
-    - {{ pin_compatible('cuda_version', max_pin='x', min_pin='x') }}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
     {% if cuda_major == "11" %}
     - cudatoolkit
     {% endif %}
     - nccl {{ nccl_version }}
     - python
-    - xgboost {{ xgboost_version }}{{ major_minor_version }}
+    - xgboost {{ xgboost_version }}
+    - _py-xgboost-mutex=*=rapidsai*
```

Comment:
Think we can drop this as the `build/string` constraint above should capture this same information

```suggestion
```